00-open-a-shapefile.Rmd

```diff
@@ -8,9 +8,9 @@ contributors: [Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
 packagesLibraries: [rgdal, raster]
-category: [self-paced-tutorial]
+category: 
```

Comment:
@mjones01  please replace - this is the correct category for this page.

    category: [self-paced-tutorial]